Sources/PointFree/EnvVars.swift

```diff
@@ -1,16 +1,19 @@
 import Foundation
 
 enum EnvVars {
-  private static var env: [String: String] {
-    return ProcessInfo.processInfo.environment
-  }
+  private static let env: [String: String] = {
+    let envFilePath = URL(fileURLWithPath: #file)
+      .deletingLastPathComponent()
+      .appendingPathComponent(".env")
 
-  static let appSecret = env["APP_SECRET"] ?? "deadbeefdeadbeefdeadbeefdeadbeef"
+    let localEnvVars = (try? Data(contentsOf: envFilePath))
+      .flatMap { try? JSONSerialization.jsonObject(with: $0) }
```

Comment:
Could use the new codable stuff here:

``` swift
JSONDecoder().decode([String: String].self, from: $0)
```

Reply:
oh jeez, duh! why do I keep reaching for `JSONSerialization`...